librarian.js

```diff
@@ -292,12 +298,159 @@ program
 
     buildInfo.notes = options.notes ? options.notes : "";
     buildInfo.branch = options.branch ? options.branch : "";
-    buildInfo.public = options.public ? true : false;
+    buildInfo.public = !!options.public;
     buildInfo.platform = platform;
 
     await addBuild(preferences, buildInfo);
-    printHeader("Build Added Successfully!")
+    printHeader("Build Added Successfully!");
+    await checkForUpdate(preferences);
+    process.exit(0);
+  });
+
+/**
+ * prerequisites :
+ *  1. Aws secret key and secret id with s3 (iam read and write access.)
+ *  2. Either self provided s3 bucket name.( or the iam access should have permission to create bucket. --> not doing this for now.)
+ *  3. We can then provide a flag for delete file if required.
+ */
+
+const uploadFile = async (s3Client, filePath, bucketName) => {
+  try {
+    // Read content from the file
+    const fileContent = fs.readFileSync(filePath);
+    let fileName = path.basename(filePath);
+
+    // Setting up S3 upload parameters
+    const params = {
+      Bucket: bucketName,
+      Key: fileName, // File name you want to save as in S3
+      Body: fileContent,
+      Acl: "public-read"
+    };
+
+    // Uploading files to the bucket
+    let data = await s3Client.upload(params).promise();
+    log(`File uploaded successfully. ${data.Location}`);
+    return data.Location;
+  } catch (e) {
+    throw e;
+  }
+};
+
+program
+  .command('submitS3 <pathToFile>')
```

Comment:
How does this new command work. Is the person supposed to call librarian `submit`, or `submitS3`? 

Instead of duplicating all of the submit code, we can also add a flag to existing submit code to upload to S3? Something like `librarian submit --s3`